src/Microsoft.OpenApi.OData.Reader/OpenApiConvertSettings.cs

```diff
@@ -179,6 +179,11 @@ public string PathPrefix
         /// </summary>
         public bool ShowMsDosGroupPath { get; set; } = true;
 
+        /// <summary>
+        /// Gets/sets links to external documentation for operations
+        /// </summary>
+        public bool ShowExternalDocs { get; set; } = true;
```

Comment:
this could be a getter that checks the configuration object I was mentioning in the other comment is not null. This way you don't have to update the code all over.

Reply:
@baywet Wouldn't that mean this setting will then be read-only?

Reply:
yeah and people would enable it by setting the configuration object with all the links, and disable it by setting that configuration object to null. What do you think?

Reply:
The scenario I'm thinking about is DevX API where we are generating Open API docs for multiple use cases. Being able to update the setting ShowExternalDocs would be much easier to do than adding or removing the LinkRelMapping setting. Also, it's more intuitive to update ShowExternalDocs than LinkRelMapping if you want to show links to external docs or not.